src/config/selectors.js

```diff
@@ -0,0 +1,73 @@
+// id selectors, mainly used for tests
```

Comment:
Make sure the ID naming convention is consistent. Either `camelCase` or `-`.

Reply:
> Make sure the ID naming convention is consistent. Either `camelCase` or `-`.

(kebab-case)